Drivers/Src/MotorController.c

```diff
@@ -2,6 +2,10 @@
 #include "CarState.h"
 #define MOTOR_DRIVE 0x221
 #define MAX_CAN_LEN 8
+#define maskMotorTempErr=1<<6; //check if motor temperature is an issue on bit 6
```

Comment:
Since these are constants, they should be defined in all caps e.g. MASK_MOTOR_TEMP_ERR.